### PR TITLE
Laser creating

### PR DESCRIPTION
**Pull request by SytzeAndr**
the shark shoots lasers. These appear on the screen. When the sharks ammo is empty, the shark can't fire more lasers. Default is at 10.

### CODE REVIEW

**Comment by MichielvdBerg** [comment]
Looks good, I approve!

**Comment by Exclaminator** [comment]
Please make sure bullets are not swimmers, this seems like an inproper name for this.

File: src/main/java/lasersharks/LaserBullet.java [hunk 1]
```diff
@@ -0,0 +1,56 @@
+package lasersharks;
+
+/**
+ * This class represents a laser projectile.
+ * @author sytze
+ *
+ */
+public class LaserBullet extends Fish {
```
Comment (Exclaminator) [inline src/main/java/lasersharks/LaserBullet.java]: Same comment

**Comment by spiderbiggen** [comment]
The names of variables in the methods do not correspond with the class names nor does the javadoc

**Pull request merged**